.gitignore

```diff
@@ -8,5 +8,6 @@
 /doc
 /.yardoc
 gemfiles/*.lock
+gemfiles/.bundle/
```

Comment:
FYI: `bundle exec appraisal install` create this directory but looks like it should be excluded from repository.

Reply:
:+1: